community/kernel/src/main/java/org/neo4j/kernel/impl/annotations/AnnotationProcessor.java

```diff
@@ -40,25 +36,11 @@
 import javax.tools.FileObject;
 import javax.tools.StandardLocation;
 
-import static org.neo4j.kernel.impl.util.Charsets.UTF_8;
 import static org.neo4j.io.fs.FileUtils.newFilePrintWriter;
+import static org.neo4j.kernel.impl.util.Charsets.UTF_8;
```

Comment:
@nawroth there is no more `org.neo4j.kernel.impl.util.Charsets.UTF_8` just change those to `java.nio.charset.StandardCharsets.UTF_8`